src/node_jsvmapi.h

```diff
@@ -250,6 +260,11 @@ NODE_EXTERN void napi_wrap(napi_env e, napi_value jsObject, void* nativeObj,
                            napi_destruct napi_destructor,
                            napi_weakref* handle);
 NODE_EXTERN void* napi_unwrap(napi_env e, napi_value jsObject);
+
+NODE_EXTERN int napi_get_internal_field_count(napi_env env, napi_value jsObject);
+NODE_EXTERN void* napi_get_internal_field_pointer(napi_env env, napi_value jsObject, int index);
+NODE_EXTERN void napi_set_internal_field_pointer(napi_env env, napi_value jsObject,  int index, void* nativeObj);
```

Comment:
I made a comment above raising a similar concern.  We should see how we can avoid use of the internal fields as our first choice.